packages/components/src/custom-gradient-picker/style.scss

```diff
@@ -1,15 +1,20 @@
 $components-custom-gradient-picker__padding: $grid-unit-20; // 48px container, 16px handles inside, that leaves 32px padding, half of which is 1å6.
 
+.components-custom-gradient-picker {
+	&:not(.is-next-has-no-margin) {
+		margin-top: $grid-unit-15;
+		margin-bottom: $grid-unit-30;
```

Comment:
`$grid-unit-30` here is the `$grid-unit-20` from `components-custom-gradient-picker__ui-line` plus the `$grid-unit-10` from the bottom margin on AnglePickerControl.

Reply:
We could've said `margin-bottom: $grid-unit-20` on L6 and passed `__nextHasNoMarginBottom={ __nextHasNoMarginBottom }` to AnglePickerControl, and it would have the same effect. I just merged those two things into a single `margin-bottom: $grid-unit-30` back compat at the top-level.